pkg/pricer/pricer.go

```diff
@@ -189,6 +220,16 @@ func (s *Pricer) getPricingPeer(peer swarm.Address) (*pricingPeer, error) {
 	return peerData, nil
 }
 
+func (s *Pricer) storePriceTable(peer swarm.Address, priceTable []uint64) error {
+	s.logger.Debugf("Storing pricetable %v for peer %v", priceTable, peer)
+	err := s.store.Put(peerPriceTableKey(peer), priceTable)
+	if err != nil {
+		s.logger.Warningf("failed to persist pricetable for peer %v: %w", peer, err)
+		return ErrPersistingBalancePeer
```

Comment:
you're swallowing debug information here. I think that `ErrPersistingBalancePeer` is not needed. This method also just wraps one method call (Put). I would consider removing it and just inlining these extra 3 lines of code wherever this is used.